internal/controllers/project/project_reconciler_function.go

```
@@ -292,3 +317,31 @@ func (t *task) removeFinalizer() {
 		t.project.GetMetadata().SetFinalizers(list)
 	}
 }
+
+// updateCondition updates or creates a condition with the specified type, status, reason, and message.
+func (t *task) updateCondition(conditionType privatev1.ProjectConditionType, status privatev1.ConditionStatus,
```

golangci-lint (2.12.2)

[error] func (*task).updateCondition is unused

(unused)